packages/orchestrator-ui-components/src/messages/useGetTranslationMessages.ts

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from 'react';
+
+import { AbstractIntlMessages } from 'next-intl';
+
+import { useAxiosApiClient } from '../components/WfoForms/useAxiosApiClient';
+import { Locale } from '../types';
+import enGB from './en-GB.json';
+import nlNL from './nl-NL.json';
+
+export type TranslationMessagesMap = Map<Locale, AbstractIntlMessages>;
+
+export const useGetTranslationMessages = (locale: string | undefined) => {
+    const apiClient = useAxiosApiClient();
+    const [backendMessages, setBackendMessages] = useState<
+        Record<string, string>
+    >({});
+
+    useEffect(() => {
+        apiClient
+            .axiosFetch<{ forms: { fields: Record<string, string> } }>(
```

Comment:
Is it possible to not use this axios client outside the forms scope? The client is even directly imported from forms. It would be hard(er) to get rid of this technical debt.